RPG/Src/XCOM2RPGOverhaul/Classes/X2TemplateHelper_RPGOverhaul.uc

```diff
@@ -459,16 +462,21 @@ static function PatchWeapons()
 						WeaponTemplate.NumUpgradeSlots = default.DefaultWeaponUpgradeSlots;
```

Comment:
While you at it you could create a config for upgrades named
`bPatchDefaultWeaponUpgradeSlots`
and make all patches of that conditionally like 
```
if (default.bPatchDefaultWeaponUpgradeSlots)
{
   WeaponTemplate.NumUpgradeSlots = default.DefaultWeaponUpgradeSlots;
}
```